scripts/convertPrefs.js

```diff
@@ -25,14 +25,14 @@ var fs = require("fs"),
 
 var inputDir = process.argv[2];
 var targetDir = process.argv[3];
+var prefsSafeType = process.argv[4];
```

Comment:
Either check `prefsSafeType` must be provided or change this line to
```
var prefsSafeType = process.argv[4] || "user";
```
to set "user" as the default if this argument is optional.

Reply:
I've modified the code to use the "or" form.  Still ... shouldn't we check that it's either "snapset" or "user" and quit if it's neither -- what if someone entered "group" for the type, for example?  Or, will there be other types of prefsSafes in the future?